make dist not working

It seems that `make dist` is not working properly. After successfully building the package using `make`, it errors out on `make dist` with
```
make[4]: *** No rule to make target 'bliss_bignum.hh', needed by 'distdir-am'.  Stop.
```
I can't locate `bliss_bignum.hh` in the tree either.

After revision https://github.com/igraph/igraph/commit/e354f8db3403b1d56ff9235c492a31a46ac6b2ff and trying to build the resulting tarball I get

```
structural_properties.c:46:10: fatal error: structural_properties_internal.h: No such file or directory
 #include "structural_properties_internal.h"
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -36,7 +36,7 @@ install-info:
 dist-hook:
 	rm -rf `find $(distdir)/examples -type d -name .arch-ids`
 
-MAINTAINERCLEANFILES = Makefile.in
+MAINTAINERCLEANFILES = Makefile.in aclocal.m4
 
 ## to make sure make deb will generate Debian packages
 .PHONY: framework msvc parsersources

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -281,7 +281,8 @@ HEADERS_PRIVATE = atlas-edges.h \
 		scg_headers.h igraph_hacks_internal.h triangles_template.h \
 		triangles_template1.h maximal_cliques_template.h prpack.h \
 		igraph_cliquer.h cliquer/graph.h cliquer/cliquer.h cliquer/misc.h \
-		cliquer/cliquerconf.h cliquer/reorder.h cliquer/set.h
+		cliquer/cliquerconf.h cliquer/reorder.h cliquer/set.h \
+		structural_properties_internal.h
 
 HEADERS_PUBLIC =../include/igraph.h 		../include/igraph_memory.h    \
 		../include/igraph_random.h 	../include/igraph_types.h     \

--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -23,7 +23,7 @@ TESTSUITE_AT = \
 	other.at foreign.at conversion.at flow.at community.at eigen.at \
 	cliques.at attributes.at arpack.at bipartite.at centralization.at \
 	version.at separators.at hrg.at microscopic.at mt.at random.at scg.at \
-	matching.at qsort.at
+	matching.at qsort.at coloring.at embedding.at
 
 TESTSUITE = testsuite
 

